packages/zmarkdown/config/remark.js

```diff
@@ -146,6 +146,7 @@ const remarkConfig = {
       'x(': '/static/smileys/pinch.png',
       '>_<': '/static/smileys/pinch.png',
       '&lt;_&gt;': '/static/smileys/pinch.png',
```

Comment:
Could we remove this now?

Reply:
Well... That's backward compatibility stuff.

Reply:
Yeah, when we started this project we decided to drop support for a bunch of bad features and bugs used as features. That's one of them, we don't care about backward compatibility here.

Reply:
OK.
I think `x(` and `&lt;_&gt;` should be removed, but `>_<` should be kept because it was the official way of writing that smiley. `X/` is the new way of doing it.

Reply:
To me x( makes sense. Let's only remove the weird html artifact that creeped in here because of an old bug in our editor.